fmf/base.py

```diff
@@ -588,3 +590,80 @@ def copy(self):
         duplicate = copy.deepcopy(self)
         self.parent = duplicate.parent = original_parent
         return duplicate
+
+    def __get_raw_data_node(self):
```

Comment:
All internal methods use a single leading underscore, let's keep this consistent.